Source/Core/Core/HotkeyManager.cpp

```diff
@@ -266,9 +258,12 @@ std::string HotkeyManager::GetName() const
 
 void HotkeyManager::GetInput(HotkeyStatus* const kb)
 {
-	for (int set = 0; set < 6; set++)
+	for (int set = 0; set < 3; set++)
```

Comment:
Not in the style guide, but I'd personally prefer curly braces on the outer block (in this case, the ```for``` loop) too if the inner block needs them for whatever reason. Any comments on that (@Lioncash perhaps)?